pandas/tests/frame/test_dtypes.py

```diff
@@ -249,27 +249,6 @@ def test_str_to_small_float_conversion_type(self):
         expected = pd.DataFrame(col_data, columns=["A"], dtype=float)
         tm.assert_frame_equal(result, expected)
 
-    @pytest.mark.parametrize(
-        "convert_integer, expected", [(False, np.dtype("int32")), (True, "Int32")]
```

Comment:
and there are no more astype's in this file? at all? i find it hard to believe we have exactly 1 test for astype on a frame

Reply:
My bad, poorly titled PR.

DataFrame.astype tests are already in test_astype.  This is our lone test for `DataFrame.convert_dtypes` being moved to test_convert_dtypes